components/site/src/lib.rs

```diff
@@ -445,6 +446,26 @@ impl Site {
         html
     }
 
+    /// Minifies html content
+    fn minify(&self, mut html: String) -> Result<String> {
+        if self.config.minify_html {
```

Comment:
can you reverse the condition so it's not indented? eg `if not self.config.minify_html { return Ok(html) } ..`